plugins/catalog-backend-module-aws/src/processors/AwsOrganizationCloudAccountProcessor.test.ts

```diff
@@ -59,11 +59,11 @@ describe('AwsOrganizationCloudAccountProcessor', () => {
               'amazonaws.com/arn':
                 'arn:aws:organizations::192594491037:account/o-1vl18kc5a3/957140518395',
               'amazonaws.com/account-id': '957140518395',
+              'amazonaws.com/account-email': 'aws-test-account@backstage.io',
               'amazonaws.com/organization-id': 'o-1vl18kc5a3',
             },
             labels: {
               'amazonaws.com/account-status': 'active',
-              'amazonaws.com/account-email': 'aws-test-account@backstage.io',
```

Comment:
btw, just thinking on the bug report on this one, if the e-mail address with `@` was already in the test data, but the previous tests didn't fail, then we aren't actually testing for the failure condition?  may not be possible in this PR, but seems like we need an extra test for the import for whatever failed to really prove this going forward. (and future proof other such label changes?). Just a thought!